stripe/src/main/java/com/stripe/android/view/AddPaymentMethodContract.kt

```diff
@@ -0,0 +1,24 @@
+package com.stripe.android.view
+
+import android.content.Context
+import android.content.Intent
+import androidx.activity.result.contract.ActivityResultContract
+
+internal class AddPaymentMethodContract :
+    ActivityResultContract<AddPaymentMethodActivityStarter.Args, AddPaymentMethodActivityStarter.Result>() {
+
+    override fun createIntent(
+        context: Context,
+        input: AddPaymentMethodActivityStarter.Args?
+    ): Intent {
+        return Intent(context, AddPaymentMethodActivity::class.java)
+            .putExtra(ActivityStarter.Args.EXTRA, input)
+    }
+
+    override fun parseResult(
+        resultCode: Int,
```

Comment:
Looks like this can be removed

Reply:
This is part of the `ActivityResultContract` interface

Reply:
Ah didn't realize it wasn't our contract, LGTM 👍